### PR TITLE
[MIRROR] A red spy has entered the base: Adds Spies, a roundstart antagonist inspired by Goonstation's Spy-Thief

### PR DESCRIPTION
**Pull request by SkyratBot**
Original PR: https://github.com/tgstation/tgstation/pull/81231
--------------------
# Disclaimer: No Goon code was referenced or used in the making of this PR 

## About The Pull Request

[Design Document (Read this for more information)](https://hackmd.io/@ L9JPMsZhRO2wI25rNI6GYg/rkYKM9Yc6)

This PR adds Spies as a new roundstart antagonist type, inspired by Spy-Thiefs from Goonstation.

Spies are tasked with stealing various objects around the station, from insulated gloves to the black box, from the clown's left leg to the bridge's communications console. 

For every item stolen, the Spy is rewarded with a random item from the Syndicate Uplink, plus some items uniquely available to the Spy. Stolen items are then shipped off and sold on the Black Market Uplink, allowing the crew - or maybe some other evil-doers - to get their hands on them. 

image

More ideas for theft items and bounties are welcome. 

## Why It's Good For The Game

See the design document for more information. 

In short: Adds a solo antagonist which has less impact than your Traitors and Heretics, but more impact than Paradox Clones and Thieves. In other words: On the same tier as old traitors. 

Seeks to embrace the sandbox aspect of antagonists more by having no precise greentext objective, and instead some suggestions for chaos you can embark in. Have fun with it!

## Changelog

:cl: Melbert
add: Spies may now roam the halls of Space Station 13. Watch your belongings closely. 
/:cl:

### CODE REVIEW

File: code/__DEFINES/role_preferences.dm [hunk 1]
```diff
@@ -16,10 +16,14 @@
...
 #define ROLE_TRAITOR "Traitor"
 #define ROLE_WIZARD "Wizard"
+<<<<<<< HEAD
 // SKYRAT EDIT START
 #define ROLE_ASSAULT_OPERATIVE "Assault Operative"
 #define ROLE_OPFOR_CANDIDATE "OPFOR Candidate"
 // SKYRAT EDIT END
+=======
+#define ROLE_SPY "Spy"
+>>>>>>> 977799a2e75 (A red spy has entered the base: Adds Spies, a roundstart antagonist inspired by Goonstation's Spy-Thief (#81231))
```
Comment (softcerv) [inline code/__DEFINES/role_preferences.dm]:
```suggestion
#define ROLE_SPY "Spy"
// SKYRAT EDIT START
#define ROLE_ASSAULT_OPERATIVE "Assault Operative"
#define ROLE_OPFOR_CANDIDATE "OPFOR Candidate"
// SKYRAT EDIT END
```

File: code/__DEFINES/role_preferences.dm [hunk 2]
```diff
@@ -145,9 +149,13 @@ GLOBAL_LIST_INIT(special_roles, list(
 	ROLE_REV_HEAD = 14,
 	ROLE_TRAITOR = 0,
 	ROLE_WIZARD = 14,
+<<<<<<< HEAD
 	// SKYRAT EDIT ADDITION
 	ROLE_ASSAULT_OPERATIVE = 14,
 	// SKYRAT EDIT END
+=======
+	ROLE_SPY = 0,
+>>>>>>> 977799a2e75 (A red spy has entered the base: Adds Spies, a roundstart antagonist inspired by Goonstation's Spy-Thief (#81231))
```
Comment (softcerv) [inline code/__DEFINES/role_preferences.dm]:
```suggestion
	ROLE_SPY = 0,
	// SKYRAT EDIT ADDITION
	ROLE_ASSAULT_OPERATIVE = 14,
	// SKYRAT EDIT END
```

**Review by jjpark-kb — APPROVED** [review]
god no, I'll deal with it later!
